Misc/NEWS.d/next/Library/2019-09-03-15-45-19.bpo-36267.z42Ex7.rst

```diff
@@ -0,0 +1 @@
+Fix IndexError in argparse when instance of  :class:`ArgumentParser` parse args like '-a=' what have a 'store_true' action.
```

Comment:
```suggestion
Fix IndexError in :class:`argparse.ArgumentParser` when a `store_true` action is given an explicit argument.
```

Reply:
Thanks for your comment. your comment looks more simple and exact.

PS: I have forget this PR's detail in fact~.